src/web/utils/AppVisibilityUtils.ts

```diff
@@ -0,0 +1,96 @@
+/**
+* AppVisibilityUtils.ts
```

Comment:
This is so much cleaner and simpler than the ifvisible implementation! Thanks for doing this.

Reply:
yes i agree. No problem, thanks for pointing me to it. :)